src/stream/src/executor/dml.rs

```diff
@@ -123,9 +124,16 @@ impl DmlExecutor {
                     }
                     yield msg;
                 }
-                Either::Right(chunk) => {
+                Either::Right(txn_msg) => {
                     // Batch data.
-                    yield Message::Chunk(chunk.chunk);
+                    match txn_msg {
+                        TxnMsg::Begin(_) | TxnMsg::End(_) => {
+                            // TODO(dylan): process txn
+                        }
+                        TxnMsg::Data(_, chunk) => {
+                            yield Message::Chunk(chunk);
+                        }
+                    }
```

Comment:
The `DmlExecutor` can receive transaction messages from batch queries. If we just yield the chunk from `TxnMsg::Data`, it should behave the same way as before.